.eslintrc.cjs

```diff
@@ -3,7 +3,10 @@ module.exports = {
     node: true,
     es2021: true,
   },
-  extends: 'eslint:recommended',
+  extends: [
+    'eslint:recommended',
+    'plugin:node/recommended',
+  ],
   parserOptions: {
     ecmaVersion: 12,
     sourceType: 'module',
```

Comment:
The [`parserOptions`](https://eslint.org/docs/user-guide/configuring/language-options#specifying-parser-options) defines how to parse the code - sometimes we have extensions required, such as `jsx` or to specify `sourceType: 'module'` to say we want to use `import/export` syntax rather than `require`.